AT/VR-AT-0006/rule.json

```diff
@@ -29,7 +29,7 @@
 			{
 				"if": [
 					{
-						"<=": [
+						"==": [
```

Comment:
Should be `===`!

Reply:
THX (I was sure, I tried that the other day ... ;-)